### ISSUE TITLE
Logger output

### ISSUE THREAD

**Issue by ebilal** [facebookresearch/XLM#209]
What are the values printed during training?

INFO - 10/04/19 14:41:25 - 0:11:11 -    1135 -   56.12 sent/s -  2143.80 words/s - MLM-en:  7.7416 -  - model LR: 3.8795e-06
INFO - 10/04/19 14:41:27 - 0:11:14 -    1140 -   56.10 sent/s -  2092.52 words/s - MLM-en:  7.6028 -  - model LR: 3.8962e-06
INFO - 10/04/19 14:41:30 - 0:11:17 -    1145 -   55.89 sent/s -  2171.22 words/s - MLM-en:  7.5921 -  - model LR: 3.9129e-06
INFO - 10/04/19 14:41:33 - 0:11:20 -    1150 -   55.94 sent/s -  2176.00 words/s - MLM-en:  7.6046 -  - model LR: 3.9295e-06

Is MLM-en the cross entropy loss? How about model LR? Thanks

**Comment by glample** [comment]
Yes, `MLM-en` is the cross-entropy loss. The `model LR` is the learning rate of the model (we found it useful to print it for debug purposes, like to see the current learning rate when the model starts overfitting / diverging, etc.).
